pkg/clusterresourceoverride/resourceoverride_store_test.go

```
@@ -0,0 +1,509 @@
+		json.Unmarshal(selectorBytes, &selectorMap)
+		obj["spec"].(map[string]interface{})["podSelector"] = selectorMap
+	}
+	return &unstructured.Unstructured{Object: obj}
+}
+
+func newTestLister(objects ...*unstructured.Unstructured) *resourceOverrideLister {
+	indexer := cache.NewIndexer(cache.MetaNamespaceKeyFunc, cache.Indexers{
+		cache.NamespaceIndex: cache.MetaNamespaceIndexFunc,
+	})
+	for _, obj := range objects {
+		indexer.Add(obj)
```

golangci-lint (2.12.2)

[error] Error return value of `indexer.Add` is not checked

(errcheck)